### PR TITLE
[tempo-distributed] Querier readiness probe

### PR DESCRIPTION
**Pull request by coufalja**
Querier is missing the readiness probe which causes the downtime of query path each time the deployment is rolled out.

### CODE REVIEW

**Comment by zalegrala** [comment]
This looks good to me, thanks for the contribution.  Can you bump the version to make CI happy?

**Comment by coufalja** [comment]
Sure thing 👍

**Comment by zalegrala** [comment]
I failed to come back to github soon enough last week.  Going to take https://github.com/grafana/helm-charts/pull/2560 first, and then you'll need to bump the version again after rebase.  I'll keep an eye today and will get this in quickly after the update.

File: charts/tempo-distributed/templates/querier/deployment-querier.yaml [hunk 1]
```diff
@@ -87,6 +87,8 @@ spec:
           securityContext:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          readinessProbe:
+            {{- toYaml .Values.tempo.readinessProbe | nindent 12 }}
```
Comment (zanhsieh) [inline charts/tempo-distributed/templates/querier/deployment-querier.yaml:91]:
```suggestion
          {{- with .Values.tempo.readinessProbe }}
          readinessProbe:
            {{- toYaml . | nindent 12 }}
          {{- end }}
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**